ros_gz_bridge/src/bridge_config.cpp

```diff
@@ -85,11 +91,20 @@ std::optional<BridgeConfig> parseEntry(const YAML::Node & yaml_node)
     return {};
   }
 
-  if (ros_type_name.empty() || gz_type_name.empty()) {
-    RCLCPP_ERROR(
-      logger,
-      "Could not parse entry: both %s and %s must be set", kRosTypeName, kGzTypeName);
-    return {};
+  if (service_name.empty()) {
+    if (ros_type_name.empty() || gz_type_name.empty()) {
+      RCLCPP_ERROR(
+        logger,
+        "Could not parse entry: both %s and %s must be set", kRosTypeName, kGzTypeName);
+      return {};
+    }
+  } else {
+    if (gz_req_type_name.empty() || gz_rep_type_name.empty()) {
+      RCLCPP_ERROR(
+        logger,
+        "Could not parse entry: both %s and %s must be set", kRosTypeName, kGzTypeName);
```

Comment:
Should these be `kGzReqTypeName` and `kGzRepTypeName`?

Reply:
Thank you for the catch